__init__.py

```diff
@@ -1,6 +1,6 @@
 # File: __init__.py
 #
-# Copyright (c) 2017-2021 Splunk Inc.
+# Copyright (c) 2017-2022 Splunk Inc.
```

Comment:
All Copyright years are manually updated. It's updated to 2022 b/c we're making change in this year!